tests/test_averager2d.py

```diff
@@ -285,17 +292,24 @@ def testAverager(self):
             self.basis,
             self._getSrc(),
             n_angles=self.n_search_angles,
-            radius=2,
+            radius=3,
```

Comment:
2 wasn't quite enough when using a circular disk instead of a (2x2) box (2root(2)), 3 seems to work well enough.